src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -196,6 +196,7 @@ const WorkflowView = _.flow(
 
     this.state = {
       activeTab: 'inputs',
+      includeOptionalInputs: false,
```

Comment:
Should this default to `true` if there are _only_ optional inputs? Otherwise, all you see is the table header. Another option would be to suppress the table altogether if there are only optional inputs.

Reply:
Related, I made an issue https://github.com/DataBiosphere/saturn-ui/issues/922 to decide whether to suppress the table altogether if there are no inputs or outputs, or at least have a message.

Reply:
Agreed that there should be some kind of message to indicate why there's only the table header (or no table if that's the route we decide) if all the inputs are optional.